packages/sanity/src/_internal/cli/actions/build/buildAction.ts

```diff
@@ -58,8 +59,40 @@ export default async function buildSanityStudio(
     flags['auto-updates'] ||
     (cliConfig && 'autoUpdates' in cliConfig && cliConfig.autoUpdates === true)
 
+  // Get the version without any tags if any
+  const coercedSanityVersion = semver.coerce(installedSanityVersion)?.version
+  if (autoUpdatesEnabled && !coercedSanityVersion) {
+    throw new Error(`Failed to parse installed Sanity version: ${installedSanityVersion}`)
+  }
+  const version = encodeURIComponent(`^${coercedSanityVersion}`)
+  const autoUpdatesImports = getAutoUpdateImportMap(version)
```

Comment:
Can we move this into the conditional below? Feels like unnecessary work and potentially throwing if the user has not opted in to auto updates

Reply:
@rexxars the reason I have it outside the conditional is because the `autoUpdatesImports` variable is used below in line `160` If I make a variable like `let autoUpdatesImports` TS can't necessarily tell if the variable is initialized so it complains. The problem with moving the whole thing after the clean is that it does into a spinner and prompt doesn't work with spinner. I think there is some opportunity to clean this up a little bit but unless you strongly feel that we should do it now, punt it for future work?

Reply:
Makes sense. Can refactor later 👍